pkg/queue/manager.go

```diff
@@ -579,6 +579,10 @@ func (m *Manager) PendingWorkloadsInfo(cqName string) []*workload.Info {
 	return cq.Snapshot()
 }
 
+func (m *Manager) ClusterQueueFromLocalQueue(lqName string) string {
+	return m.localQueues[lqName].ClusterQueue
```

Comment:
Can we be sure this will not panic? I'm wondering if there might be race conditions when `lqName` is already deleted.